_config.yml

```diff
@@ -1,4 +1,4 @@
-highlighter: rouge
```

Comment:
According to this one https://jekyllrb.com/docs/upgrading/2-to-3/#syntax-highlighter-changed, pygments is still an option.